lib/ecosystems/package.go

```diff
@@ -47,6 +47,26 @@ func GetPackageData(purl packageurl.PackageURL) (*packages.GetRegistryPackageRes
 	return resp, nil
 }
 
+func GetPackageVersionData(purl packageurl.PackageURL) (*packages.GetRegistryPackageVersionResponse, error) {
+	client, err := packages.NewClientWithResponses(server)
+	if err != nil {
+		return nil, err
+	}
+
+	// Ecosyste.ms has a purl based API, but unfortunately slower
+	// so we break the purl down to registry and name values locally
+	// params := packages.LookupPackageParams{Purl: &p}
+	// resp, err := client.LookupPackageWithResponse(context.Background(), &params)
```

Comment:
can we remove this? including the comment about the purl api? i don't find it particularly useful 😄